index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">
+          <img src="images/Union.png" alt="menubar" />
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section class="Port">
+        <div hreh="#" class="Menu"></div>
+        <h1>
+          Hey there!<br />
+          I'm Michael
+        </h1>
+
+        <h2>
+          I'm a Software<br />
+          Developer
+        </h2>
```

Comment:
Your design
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/86a6a036-472b-45a6-99d3-de989e895df9)

Figma
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/9d1bc650-a98c-4183-99ea-846371e62bbf)

Kindly increase the font size of the header `I'm a Software . . ." using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 🛠